### PR TITLE
Add new bounty and audits in progress

### PR DESCRIPTION
**Pull request by claudijd**

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -2,13 +2,19 @@
 
 ## Bug Bounty Program
 
-We operate a **[bug bounty program](https://immunefi.com/bounty/wormhole/)** to financially incentivize independent researchers (with up to $10,000,000 USDC) to find and responsibly disclose security issues in Wormhole.
+The Wormhole project operates two bug bounty programs to financially incentivize independent researchers to find and responsibly disclose security issues in Wormhole.
```
Comment (SEJeff) [inline SECURITY.md:5]:
```suggestion
The Wormhole project operates two bug bounty programs to financially incentivize independent researchers for finding and responsibly disclosing security issues.
```

File: SECURITY.md [hunk 2]
```diff
@@ -17,26 +23,26 @@ As these 3rd party audits are completed and issues are sufficiently addressed, w
...
+  - Check out places where there are unit/integration tests, but they lack sufficient [negative test](https://en.wikipedia.org/wiki/Negative_testing) coverage.
+- Review different smart contract implementations (eg. Solana, EVM, CosmWasm, Move) and attempt to understand how and why they are different.
   - Does one chain have a safety check that another chain doesn't?
   - Does one chain have a specific set of nuances / gotchas that that were missed on another chain?
 - Consider going beyond the source code
-  - Review the deployed contracts on chain. Is something odd that we missed?
+  - Review the deployed contracts on chain. Is something odd that may have been missed?
 
-We'll continue to iterate on this list of white-hat bootstrap strategies as we grow our lessons learned internally hacking on Wormhole and from other white-hats who have been successful via our bug bounty program.
+This section will continue to iterate on white-hat bootstrap strategies as lessons are learned hacking on Wormhole and from other white-hat community members.
```
Comment (SEJeff) [inline SECURITY.md:43]:
```suggestion
This section will continue iterating on white-hat bootstrap strategies as lessons are learned hacking on Wormhole and from community members.
```

Redundant use of `white-hat` is redundant.

File: SECURITY.md [hunk 3]
```diff
@@ -17,26 +23,26 @@ As these 3rd party audits are completed and issues are sufficiently addressed, w
...
 
-It's important to remember this is an iterative process. If you spend the time to come up with a new test case, but didn't actually find a bug, we'd be extremely appreciative if you'd be willing to send a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) with additional positive and negative test cases. This process has shown repeatedly to improve your ability to understand Wormhole, and will increase your odds of success.
+It's important to remember this is an iterative process and stay positive. If you spend the time to come up with a new test case, but didn't actually find a bug, please send a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) with additional positive and negative test cases. This process has shown repeatedly to improve your ability to understand Wormhole, and will increase your odds of finding future bugs.
```
Comment (SEJeff) [inline SECURITY.md:45]:
```suggestion
It's important to remember this is an iterative process and to stay positive. If you spend the time coming up with a new test case, but didn't actually find a bug, please send a [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) with additional positive and negative test cases. This process has shown repeatedly to improve your ability to understand Wormhole, and will increase your odds of finding future bugs.
```

File: SECURITY.md [hunk 4]
```diff
@@ -58,4 +64,4 @@ Additionally, ensure:
 - The Wormhole team has sufficient contact information and an associated call or page tree to reach you in the event of a security incident.
 - That Wormhole has the full upgrade authority on relevant bridge contracts to act quickly in the case of a security incident.
 - You have an established incident response program in place, with established patterns and playbooks to ensure deterministic outcomes for containment.
-- When security issues do occur, that the chain makes every attempt to inform affected parties and leads with transparency.
+- When security issues do occur, that the chain makes every attempt to inform affected parties and leads with transparency.
```
Comment (SEJeff) [inline SECURITY.md:67]: This sentence doesn't make sense. Was it supposed to say something like this?
```suggestion
- When security issues do occur, please make sure that the chain makes every attempt to inform affected parties and leads with transparency.
```

**Review by SEJeff — DISMISSED** [review]

**Review by daviesalex — APPROVED** [review]
LGTM

**Review by kcsongor — APPROVED** [review]

**Review by njkumr — COMMENTED** [review]
looks good

**Pull request merged**